src/index.js

```diff
@@ -131,8 +131,8 @@ class Service {
 
     let { query, options } = multiOptions(id, params, this.id);
 
-    // We can not update the id
-    delete data[this.id];
```

Comment:
I'd maybe put an `if` around this rather than completely remove it.

```js
if (this.id === '_id) {
  delete data[this.id];
}
```
because I'm pretty sure that mongo will bomb if you try to do an update with an `_id` and are using the standard `_id` field.